### PR TITLE
[spirv] Variable/Resource in namespaces.

### PR DESCRIPTION
**Pull request by ehsannas**

### CODE REVIEW

File: tools/clang/lib/SPIRV/TypeTranslator.cpp [hunk 1]
```diff
@@ -1266,6 +1259,40 @@ TypeTranslator::getLayoutDecorations(const DeclContext *decl, LayoutRule rule) {
...
+  }
+
+  if (shouldSkipInStructLayout(field))
+    return;
+
+  if (!isa<DeclaratorDecl>(field)) {
+    return;
+  }
+
+  (*decls).push_back(field);
```
Comment (antiagainst) [inline tools/clang/lib/SPIRV/TypeTranslator.cpp]: `decls->push_back(field)`

File: tools/clang/lib/SPIRV/TypeTranslator.cpp [hunk 2]
```diff
@@ -1266,6 +1259,40 @@ TypeTranslator::getLayoutDecorations(const DeclContext *decl, LayoutRule rule) {
...
+}
+
+const llvm::SmallVector<const Decl *, 4>
```
Comment (antiagainst) [inline tools/clang/lib/SPIRV/TypeTranslator.cpp]: `const` in the return type is redundant.

File: tools/clang/lib/SPIRV/TypeTranslator.cpp (hunk 1 above)
Comment (ehsannas) [reply tools/clang/lib/SPIRV/TypeTranslator.cpp]: Oops, yeah!

File: tools/clang/lib/SPIRV/TypeTranslator.cpp (hunk 2 above)
Comment (ehsannas) [reply tools/clang/lib/SPIRV/TypeTranslator.cpp]: Yup.

**Pull request merged**